sql-statements/sql-statement-alter-table.md

```diff
@@ -162,6 +164,8 @@ The following major restrictions apply to `ALTER TABLE` in TiDB:
 
 * Spatial data types are not supported.
 
+* `ALTER TABLE t CACHE | NOCACHE` is not a standard MySQL syntax but a TiDB extension. See [Cached Tables](/cached-tables.md) for more information.
```

Comment:
SQL is a standard, but for MySQL it is only a reference implementation. I think we can improve the readability by just calling it an extension to MySQL syntax:
```suggestion
* `ALTER TABLE t CACHE | NOCACHE` is a TiDB extension to MySQL syntax. See [Cached Tables](/cached-tables.md) for more information.
```